### ISSUE TITLE
Chrome on Android crashing when adding photo to cropper

### ISSUE THREAD

**Issue by noisecapella** [mitodl/micromasters#2028]
I think it's running out of memory? I'm not sure. The photo being uploaded is 2.4mb. The screen turns black and then I get an 'Aw snap!' crash message after a minute or so

**Comment by pdpinch** [comment]
Is this relate?

https://sentry.io/mit-office-of-digital-learning/micromasters/issues/191306984/

**Comment by noisecapella** [comment]
No, that profile is @aliceriot's and this bug is before the 'Save' button is clicked

**Comment by noisecapella** [comment]
This is an intermittent issue BTW, sometimes I can reproduce this 100% of the time and other times things just work fine. I feel like it's a memory issue fundamentally

**Comment by aliceriot** [comment]
hum not sure what we could do about this, we can't intercept the photo before the user adds it to the cropper, except maybe with some messaging that encourages them to avoid large photos or close open applications/tabs before attempting.

**Comment by noisecapella** [comment]
Maybe there are memory allocations we can avoid duplicating, though I agree it's hard to know if we can fix this. I'll bring in my Android debug cable tomorrow so I can see what the memory use is like.

**Comment by aliceriot** [comment]
@noisecapella did you have a chance to do more investigation of this?

**Comment by noisecapella** [comment]
I forgot to bring my debug cable today, I'll look at this tomorrow. I haven't seen this happen today so far FWIW
